travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,27 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
+    :return: 
+    """
+
+    """TODO: finish with interaction with API"""
+
+    response = request.GET.get('lon') + " " + request.GET.get('lat')
```

Comment:
Nope. Here is only sring for testing appereance on the frontend.
Expectedly it will return name of location received from map API